OMEdit/OMEditLIB/Annotations/BooleanAnnotation.cpp

```diff
@@ -1,19 +1,9 @@
 #include "BooleanAnnotation.h"
 
 BooleanAnnotation::BooleanAnnotation()
-  : mValue(true)
+  : DynamicAnnotation()
```

Comment:
Why call the constructor for `DynamicAnnotation` here? The default constructor will be called anyway.

Reply:
Also, if you want to avoid specifying default values twice it would probably be better to just implement `clear` as:
```
void RealAnnotation::clear()
{
  *this = {};
}
```
I.e. just assign a default-constructed instance to `*this`.